msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/local/TestCase497038.java

```diff
@@ -83,7 +82,7 @@ public void test_497038() throws Throwable {
         azureSampleApp.signInWithSingleAccountFragment(username, password, getBrowser(), true, microsoftStsPromptHandlerParameters);
 
         // sleep as it can take a bit for UPN to appear in Azure Sample app
-        Thread.sleep(TimeUnit.SECONDS.toMillis(5));
+        Thread.sleep(TimeUnit.SECONDS.toMillis(10));
```

Comment:
seems like it takes much longer

Reply:
For the upn to appear? I can increase the sleep time

Reply:
I meant you already increased it, so I assume it was taking longer than 5 seconds for you and hence you increased it to 10 seconds?

Reply:
I think this is left over code I had. from my test passes it looks like it was still failing with 10 seconds. I'll probably look into this more next sprint.